OpenOversight/app/static/css/openoversight.css

```diff
@@ -578,8 +589,7 @@ tr:hover .row-actions {
 }
 
 .officer-face {
-    min-width: 200px;
-    min-height: 200px;
+    height: 300px;
```

Comment:
With the previous setup you got these HUGE photos that threw off the flow of the page, so I set a standard height and allow for the width to go to what maintains the proper aspect ratio.